flink-kubernetes/src/test/java/org/apache/flink/kubernetes/KubernetesClusterDescriptorTest.java

```diff
@@ -102,8 +102,8 @@ public void testKillCluster() throws Exception {
 		final KubernetesClusterDescriptor descriptor = new KubernetesClusterDescriptor(FLINK_CONFIG, flinkKubeClient);
 
 		final ClusterSpecification clusterSpecification = new ClusterSpecification.ClusterSpecificationBuilder()
-			.setMasterMemoryMB(1)
-			.setTaskManagerMemoryMB(1)
+			.setMasterMemoryMB(1234)
```

Comment:
We do not need to set these options, and the default value 768 in `ClusterSpecificationBuilder ` is already suitable for the test.

Reply:
I will fix it.